specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/examples/GetMaintenanceWindowOptions.json

```diff
@@ -0,0 +1,34 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "Default-SQL-SouthEastAsia",
+    "workspaceName": "testsvr",
+    "sqlPoolName": "testdb",
+    "maintenanceWindowOptionsName": "current",
+    "api-version": "2019-06-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/Default-SQL-SouthEastAsia/providers/Microsoft.Synapse/workspaces/testsvr/sqlPools/testdb/maintenanceWindowOptions/current",
+        "name": "current",
+        "type": "Microsoft.Synapse/workspaces/sqlPools/maintenanceWindowOptions",
+        "properties": {
+          "isEnabled": true,
+          "maintenanceWindowCycles": [
```

Comment:
do you need 2 resource types? this seems to cover the maintenance window type as well.